dietstory-routing/registration/models.py

```diff
@@ -29,17 +32,44 @@ def get_db_prep_value(self, value, connection, prepared=False):
 
         if value is None:
             return DATE_FIELD_DEFAULT
+
         return value
 
 
-class Accounts(models.Model):
+class AccountsManager(BaseUserManager):
+
+    use_in_migrations = True
+
+    def _create_user(self, username, password, email, birthday, admin_level):
+        if not username:
+            raise ValueError("Username must be set.")
+        if not email:
+            raise ValueError("Email must be set.")
+        if not password:
+            raise ValueError("Password must be set")
+        if not birthday:
+            raise ValueError("Birthday must be set.")
+
```

Comment:
private functions use double underscore __ if you don't want python `dir` to pick the method up

Reply:
Good point!